bazel/envoy_build_system.bzl

```diff
@@ -0,0 +1,39 @@
+# References to Envoy external dependencies should be wrapped with this function.
+def envoy_external_dep_path(dep):
+    return "//external:%s" % dep
+
+ENVOY_COPTS = [
+    "-fno-omit-frame-pointer", "-Wall", "-Wextra", "-Werror", "-Wnon-virtual-dtor",
+    "-Woverloaded-virtual", "-Wold-style-cast", "-std=c++0x",
+    "-includesource/precompiled/precompiled.h", "-iquote", "include", "-iquote", "source"
```

Comment:
Yes, that what I've meant... But it indeed wouldn't work with `BUILD` files in subdirectories.